### DISCUSSION TITLE
react web site is slow in mobile phone

### DISCUSSION THREAD

**Discussion by BhathikaGimhan** [community/community#82711] [Programming Help]
### Body

[idea-drive.web.app](url)

this is my own project. i crate this project using ReactJS. but my web site is slow in mobile phone. please give me a solution

this is package.json 
{
  "name": "idea-drive",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@emailjs/browser": "^3.11.0",
    "@fontsource/roboto": "^5.0.8",
    "@google/generative-ai": "^0.1.1",
    "@mui/icons-material": "^5.14.16",
    "@mui/material": "^5.14.15",
    "@mui/styled-engine-sc": "^6.0.0-alpha.3",
    "@react-google-maps/api": "^2.19.2",
    "@testing-library/jest-dom": "^5.17.0",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "bootstrap": "^5.3.2",
    "flowbite": "^2.2.0",
    "jquery": "^3.7.1",
    "popper.js": "^1.16.1",
    "react-bootstrap": "^2.9.1",
    "react-helmet-async": "^2.0.4",
    "react-markdown": "^9.0.1",
    "react-router-dom": "^6.17.0",
    "react-scripts": "5.0.1",
    "react-slick": "^0.29.0",
    "slick-carousel": "^1.8.1",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "tailwindcss": "^3.3.3"
  }
}

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by faridnec** [comment]
In case you haven't done code splitting, split your JavaScript bundles into smaller files. This can be achieved using React's built-in React.lazy and Suspense or tools like Webpack or you can implement lazy loading for components that are not immediately required when the application loads. (Instead of loading the entire application or all its components when a user first visits a page, lazy loading loads certain components until they are actually needed)

but btw I checked the link you shared, it shows 404 error

**Reply by BhathikaGimhan** [reply]
my web site address is [idea Drive](https://idea-drive.web.app)
